<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormaPopis</class>
 <widget class="QMainWindow" name="FormaPopis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>919</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Popis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_28">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_27">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_26">
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_29">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_41">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 127);</string>
               </property>
               <property name="text">
                <string>NAZIV:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEdit_30">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>441</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>16</pointsize>
                </font>
               </property>
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_42">
               <property name="font">
                <font>
                 <family>Times New Roman</family>
                 <pointsize>18</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 127);</string>
               </property>
               <property name="text">
                <string>ŠIFRA:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_23" columnstretch="1,1">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_24">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="0,0,1">
                 <item>
                  <widget class="QLabel" name="label_36">
                   <property name="font">
                    <font>
                     <family>Times New Roman</family>
                     <pointsize>18</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(0, 0, 127);</string>
                   </property>
                   <property name="text">
                    <string>KOLIČINA:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_16">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>3</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_26">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Times New Roman</family>
                     <pointsize>16</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="0,0,1">
                 <item>
                  <widget class="QLabel" name="label_37">
                   <property name="font">
                    <font>
                     <family>Times New Roman</family>
                     <pointsize>18</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(0, 0, 127);</string>
                   </property>
                   <property name="text">
                    <string>CENA:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_17">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>57</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_27">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Times New Roman</family>
                     <pointsize>16</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="2" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="0,0,1">
                 <item>
                  <widget class="QLabel" name="label_38">
                   <property name="font">
                    <font>
                     <family>Times New Roman</family>
                     <pointsize>18</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(0, 0, 127);</string>
                   </property>
                   <property name="text">
                    <string>SUMA:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_18">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>57</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_28">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Times New Roman</family>
                     <pointsize>16</pointsize>
                    </font>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::NoFocus</enum>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="3" column="0">
                <widget class="QListWidget" name="listWidget_6">
                 <property name="minimumSize">
                  <size>
                   <width>267</width>
                   <height>200</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="lineWidth">
                  <number>1</number>
                 </property>
                 <property name="autoScroll">
                  <bool>true</bool>
                 </property>
                 <property name="dragDropOverwriteMode">
                  <bool>false</bool>
                 </property>
                 <property name="movement">
                  <enum>QListView::Snap</enum>
                 </property>
                 <property name="isWrapping" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="batchSize">
                  <number>50000</number>
                 </property>
                 <property name="sortingEnabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QGridLayout" name="gridLayout_25">
               <item row="0" column="0">
                <widget class="QTableView" name="tableView_6">
                 <property name="minimumSize">
                  <size>
                   <width>470</width>
                   <height>270</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="0,1">
                 <item>
                  <widget class="QLabel" name="label_39">
                   <property name="font">
                    <font>
                     <family>Times New Roman</family>
                     <pointsize>24</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(0, 0, 127);</string>
                   </property>
                   <property name="text">
                    <string>UKUPNO:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_40">
                   <property name="minimumSize">
                    <size>
                     <width>310</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Times New Roman</family>
                     <pointsize>24</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(0, 0, 127);</string>
                   </property>
                   <property name="text">
                    <string>0,00</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
        <zorder></zorder>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777169</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_29">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_43">
               <property name="maximumSize">
                <size>
                 <width>57</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Broj popisa:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_44">
               <property name="maximumSize">
                <size>
                 <width>71</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Datum popisa:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Napravi novi popis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Ažuriraj stari popis</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1229</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
